classes/task/poll_stale_conversions.php

```diff
@@ -0,0 +1,123 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A scheduled task.
+ *
+ * @package    local_smartmedia
+ * @copyright  2019 Matt Porritt <mattp@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+namespace local_smartmedia\task;
+
+use core\task\scheduled_task;
+use local_smartmedia\aws_api;
+use local_smartmedia\aws_elastic_transcoder;
+use \local_smartmedia\conversion;
+
+class poll_stale_conversions extends scheduled_task {
+
+    public function get_name() {
```

Comment:
Missing doc blocks for methods for whole class

Reply:
Meant to add them, ran out of time :)